assets/js/main.js

```diff
@@ -254,6 +252,19 @@ function directory(jQuery) {
 
     //load posters
     fetch("posters.json").then(res=>res.json()).then(data=>{
+        // Synchronously fetch the overrides, in order to modify posters before adding rows to the table
+        $.ajax({
+            async: false,
+            type: 'GET',
+            url: "posters-overrides.json",
+            success: function (overrides) {
+                data.posters.map(
+                    function (e, i) {
+                        Object.assign(e, overrides.posters[i]);
```

Comment:
this one still just uses `[i]` instead of creating a proper mapping table -- it would fail for entries which are not in overrides (e.g. software demos)